src/v/cluster/tx_registry_stm.h

```diff
@@ -13,24 +13,71 @@
 
 #include "cluster/fwd.h"
 #include "cluster/persisted_stm.h"
+#include "cluster/tx_hash_ranges.h"
 #include "model/fundamental.h"
 #include "model/record.h"
+#include "raft/consensus.h"
 #include "raft/errc.h"
 #include "raft/logger.h"
 #include "raft/state_machine.h"
 #include "utils/expiring_promise.h"
 #include "utils/mutex.h"
 
 #include <absl/container/flat_hash_map.h>
+#include <absl/container/node_hash_set.h>
 
 namespace config {
 struct configuration;
 }
 
 namespace cluster {
 
+using tm_partitions_count = int32_t;
```

Comment:
we have a `named_type` for this

Reply:
this type isnt reuse outside of tx_egistry_stm altogether so I dropped it